addons/editor-devtools/DevTools.js

```diff
@@ -2207,105 +1264,13 @@ export default class DevTools {
     this.costTab = guiTabs[1];
     this.costTabBody = document.querySelector("div[aria-labelledby=" + this.costTab.id + "]");
 
-    if (!document.getElementById("s3devFind")) {
-      // noinspection JSUnresolvedVariable
-      root.insertAdjacentHTML(
-        "beforeend",
-        `
-                <div id="s3devToolBar">
```

Comment:
I think you removed this without checking if other files reference this id.

This id should have been kept, it's referenced on Tab.js